docs/build/tutorials/wharfkit/howto_react.md

```diff
@@ -0,0 +1,231 @@
+          .catch((err: any) => {
+            console.log(err);
+          });
+      }
+    });
+  }, [])
+```
+
+The session kit can manage multiple sessions, but in our example, we will only have one session at a time, so it will be sufficient to retrieve the first element from the sessions array.
+The account name in WAX is a 64-bit number. To get its string equivalent, we need to convert it using String().
+
+[Sample Code](https://github.com/3dkrender/WAX_CS_Template/blob/main/Client/src/Pages/GetUserTokens/GetUserTokens.tsx)
+
+# Sign and Send a Transaction
+
+We've already seen how to retrieve the session from any component. Once retrieved, another operation of great interest to us is to send a transaction to the blockchain on behalf of the user.
+
+:::warning Important
+The operations that the user must sign should be requested from the client side. Our component will generate the transaction, thanks to the tools of the session kit, and this transaction will be sent to the user's wallet for signing. This means the user doesn't need to share their private key with the application at any time.
+
+If we wanted to make a transaction on behalf of the application itself, we would need to have the private key of the application's account available, which carries inherent risks. This will never be done on the client side. If our application needs to use a password or private key, it should do so from the server side, always outside the user's context.
+:::
+
+To prepare the transaction and send it to the wallet to be signed the function is
+
+```javascript
+session.transact()
+```
+
+As a parameter, we can pass an action or an array of actions, in addition to other useful options such as the wait time before considering the transaction as expired (expireSeconds) or whether we want the transaction to be published on the blockchain or just simulated (broadcast) - useful for testing purposes.
+
+In the analyzed sample code, it has been decided to assume that an array of actions (actions[]) will always be sent, with at least one action. Given that this might be a repetitive operation, the actions will be defined in the corresponding components and executed from a helper function.
+
+Definition of the actions array:
+
+```javascript
+    const trx: any = await InitTransaction({
+      actions: [{
+        account: 'eosio.token',
+        name: 'transfer',
+        authorization: [{
+          actor: userName,
+          permission: 'active',
+        }],
+        data: {
+          from: userName,
+          to: '3dkrenderwax',
+          quantity: `${value} WAX`,
+          memo: 'WAX donation',
+        },
+      }]
+    });
+```
+[Sample Code](https://github.com/3dkrender/WAX_CS_Template/blob/7b35853b5d7f44482b20a342fc27054989fff93d/Client/src/Pages/Transaction/PushTransaction.tsx#L31C10-L31C10)
+
+Creation of the transaction, sending to the wallet, and receiving the result:
+
+```javascript
+export const TAPOS = {
+  blocksBehind: 3,
```

Comment:
FYI: This isn't a valid parameter for the Session Kit's `transact` call. It'll accept it, but it doesn't do anything. By default it just uses the last irreversible block.

Reply:
Thanks. Fixed and added "chain" option